src/rgw/rgw_cache.h

```diff
@@ -176,7 +177,13 @@ class ObjectCache {
   void do_invalidate_all();
 
 public:
-  ObjectCache() : lru_size(0), lru_counter(0), lru_window(0), cct(NULL), enabled(false) { }
+  ObjectCache(CephContext* cct) 
+  : cct(cct),
+    lru_max_size(cct->_conf.get_val<int64_t>("rgw_cache_lru_size") / 2),
+    lru_window(lru_max_size / 2),
```

Comment:
you're dividing by 2 in both of these lines, but i think you only want that for lru_window

Reply:
Yes, you are correct. Modified